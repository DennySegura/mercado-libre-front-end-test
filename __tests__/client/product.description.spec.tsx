import 'jest';
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProductDescription from '../../src/app/components/product-description';

describe('Test suites for react components', () => {
  it('It should render <ProductDescription />', () => {
    const _product_description = renderer.create(<ProductDescription description={'Test'}/>);
    let product_description_tree = _product_description.toJSON();
    expect(product_description_tree).toMatchSnapshot();
  });
});
