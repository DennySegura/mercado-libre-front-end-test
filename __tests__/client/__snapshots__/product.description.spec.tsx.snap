// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suites for react components It should render <ProductDescription /> 1`] = `
<div
  className="mt column"
>
  <span
    className="description--title mb3"
  >
    Descripci√≥n del producto
  </span>
  <p
    className="description--text mb3 tc-dusty-gray"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Test",
      }
    }
  />
</div>
`;
